### PR TITLE
show plot shortcuts in sphinx docs

### LINKED ISSUE #416: Show plot shortcuts on readthedocs
## General

* pyfar version: 0.51

We want to show the shortcuts in the docs, in the ideal case using the json files where they are stored...

### PR DESCRIPTION
**Pull request by f-brinkmann**
### Which issue(s) are closed by this pull request?

Closes #416 

### Changes proposed in this pull request:

- extend `pf.plot.shortcuts` to output shortcuts in sphinx readable version
- update testing of `pf.plot.shortcuts`
- generate sphinx readable shortcuts in `docs/conf.py`
- include sphinx readable shortcuts in `docs/concepts/pyfar.plot.rst`

### CODE REVIEW

File: pyfar/plot/utils.py [hunk 1]
```diff
@@ -236,47 +247,89 @@ def shortcuts(show=True):
...
+        sc_str += ("- Moving and zooming the x and y axes is supported by all "
+                   "plots.\n"
+                   "- Moving and zooming the colormap is only supported by "
+                   "plots that have a colormap.\n"
+                   "- Toggling the x-axis is supported by: "
+                   f"{', '.join(x_toggle)}\n"
+                   "- Toggling the y-axis is supported by: "
+                   f"{', '.join(y_toggle)}\n"
+                   "- Toggling the colormap is supported by: "
+                   f"{', '.join(cm_toggle)}\n"
```
Comment (sikersten) [inline pyfar/plot/utils.py:325]: Is it possible to link here with `:py:func:`~pyfar.plot.`?

**Review by sikersten — COMMENTED** [review]
Thanks, nice improvement! Only have two small suggestions, otherwise it looks good.

File: pyfar/plot/utils.py [hunk 2]
```diff
@@ -236,47 +247,89 @@ def shortcuts(show=True):
...
+
+            if layout == "console":
+                sc_str += f'{", ".join(key)}: {p}\n'
+            else:
+                sc_str += (f'   * - {", ".join(key)}\n'
+                           f'     - :py:func:`~pyfar.plot.{p}`\n')
+
+        sc_str += ("\nNote that not all plots are available for TimeData and "
+                   "FrequencyData objects as detailed in the documentation of "
+                   "plots.\n\n")
```
[the remark this answers is not in the document]
Comment (f-brinkmann) [reply pyfar/plot/utils.py]: I linked to the plot module

File: pyfar/plot/utils.py (hunk 1 above)
Comment (f-brinkmann) [reply pyfar/plot/utils.py]: Yes, absolutlely

**Pull request merged**